src/main/java/pl/project13/core/cibuild/BambooBuildServerData.java

```diff
@@ -43,7 +43,8 @@ public static boolean isActiveServer(Map<String, String> env) {
   @Override
   void loadBuildNumber(@Nonnull Properties properties) {
     String buildNumber = Optional.ofNullable(env.get("bamboo.buildNumber"))
-            .orElseGet(() -> env.getOrDefault("BAMBOO_BUILDNUMBER", ""));
+            .or(() -> Optional.ofNullable(env.get("BAMBOO_BUILDNUMBER")))
```

Comment:
I don't think [`Optional` has an `or`-method for java 11](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html), or at least I can't find it on the man pages.

With an
```java
String buildNumber = Optional.ofNullable(env.get("bamboo.buildNumber"))
            .orElseGet(() ->
                    Optional.ofNullable(env.get("BAMBOO_BUILDNUMBER"))
                            .orElseGet(() -> env.getOrDefault("bamboo_buildNumber", ""))
            );
```
I think that statement just gets unreadable.

What about a:
```java
String buildNumber = Arrays.asList(
            env.get("bamboo.buildNumber"),
            env.get("BAMBOO_BUILDNUMBER"),
            env.get("bamboo_buildNumber")
    ).stream()
            .filter(Objects::nonNull)
            .findFirst()
            .orElse("");
```
?

Reply:
Nevermind. Ignore my comment. I still had a java 1.8 reference on my path causing a failure. In fact there is a [or](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html#or(java.util.function.Supplier))-method available. So let me go ahead and merge.

Thanks again :-)

Reply:
No problem. The or-method is available from Java 9 and up. Thanks for merging 👍  Could you make an estimate when this change will be making its way into [git-commit-id-maven-plugin](https://github.com/git-commit-id/git-commit-id-maven-plugin)?

Reply:
Thanks for letting me know since when the method is available!
Good question. I initially wanted to tackle https://github.com/git-commit-id/git-commit-id-maven-plugin/pull/535 and need to understand in detail how this relocation stuff works before I cut a release. Given that my free time to work on this project is now very limited again it would probability make sense to cut a release without those changes (so this just leaves the relocation). If I'm optimistic I might find time on the next weekend, so yeah maintainers welcome ;-)

Reply:
Version 5.0.0 of the maven plugin is released and available via: https://repo1.maven.org/maven2/io/github/git-commit-id/git-commit-id-maven-plugin/5.0.0/

Reply:
Hi there, just tested the new version and it works 😄  thank you!

Reply:
Thanks for confirming that everything works fine now!